.github/workflows/backport.yaml

```
@@ -0,0 +1,58 @@
+      github.event.issue.pull_request.merged_at != null &&
+      startsWith(github.event.comment.body, '@khulnasoft-bot backport release/')
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Extract branch name
+        run: |
```

actionlint (1.7.4)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)